server/routes/creatingLicences/handlers/caseload.test.ts

```diff
@@ -47,9 +44,9 @@ describe('Route Handlers - Create Licence - Caseload', () => {
       expect(res.render).toHaveBeenCalledWith('pages/create/caseload', {
         caseload: [
           {
-            name: 'Jones',
+            name: 'Joe Rogan',
             crnNumber: 'X381306',
-            conditionalReleaseDate: '03 August 2022',
+            conditionalReleaseDate: '09/09/2022',
           },
```

Comment:
Might be some content challenge on date formats - but not sure we're had a steer on consistent date format yet!